beaker-tests/Sanity/copr-cli-basic-operations/runtest-dnf-copr.sh

```diff
@@ -46,24 +46,26 @@ rlJournalStart
         OUTPUT=`mktemp`
 
         # Test `dnf copr search' command
+        # TODO Implement the search command in DNF5
```

Comment:
Again, do we need this in DNF5? If yes, open an issue, if not - let's just leave it with comment that we don't plan to implement it.

Reply:
I don't know if we necessarily need it, but in comparison to the not-very-useful `--available-by-user` parameter, the search command could be nice to have. I will create a ticket for that. Do you prefer it in the DNF or Copr issue tracker?